.travis.yml

```diff
@@ -49,18 +52,26 @@ install:
   - pip install -r requirements.txt
   - pip install .
   - pip install coveralls
+  # install vaxrank and its dependent data
+  - pip install --upgrade -r $PWD/vaxrank/requirements.txt
+  - pyensembl install --release 87 --species human
```

Comment:
@iskandr I have a vague memory of you caching the Ensembl data in other Travis configs, did that ever work out?